lighthouse-core/report/html/renderer/pwa-category-renderer.js

```diff
@@ -46,6 +46,42 @@ class PwaCategoryRenderer extends CategoryRenderer {
     return categoryElem;
   }
 
+  /**
+   * @param {LH.ReportResult.Category} category
+   * @return {DocumentFragment}
+   */
+  renderScoreGauge(category) {
+    // Defer to parent-gauge style if category error.
+    if (category.score === null) {
```

Comment:
I can't decide if I like the explicit documentation that `score === null` when it fails or would prefer to play it safe with `Number.isFinite`, any insight you used to share with me?

Reply:
> any insight you used to share with me?

I used it just because that's what `categoryRenderer.renderScoreGauge` uses :)

Looking more into it now, `null` is what `ReportScoring.arithmeticMean` returns in case of error, and combined with the fact that we haven't seen any issues with it for plain old `categoryRenderer`...seems fine?